### DISCUSSION TITLE
Unfortunately, we were unable to verify your academic affiliation based on the information you submitted

### DISCUSSION THREAD

**Discussion by superrhodes** [community/community#51490] [GitHub Education]
I am a high school student, and I have attempted to sign up for the Github Student Developer Pack 8 times. I have uploaded a scanned picture of my student id, normal picture of my student id, two pictures of my academic transcript and a picture of my county's grading portal. I have put my full name and a "short bio" on my user profile. I have tried to use my school email, but I can't verify it due to the filters on school emails. I have looked at similar discussions and have tried the suggestions in them. At this point, I don't know what to do. I want to use Github Pro so I can use Codespaces for longer as I use them to run Python when I am away from my personal computer, like when I am at school. Does anybody have any suggestions as to what I should do at this point?

**Comment by samus-aran** [comment]
Hi @superrhodes  👋

Thank you for reaching out to us. We've raised this with the internal team and would like to ask you to re-apply. Let us know how you get on. 👍

**Reply by superrhodes** [reply]
When I try to apply using my school, it asks for an email address from kmhsmagnet.com. We were never given an email like that, we just use our county ones, which I still can't use right now as it filters emails from Github. I have tried to contact my school's IT department on the filter, but they haven't responded yet and as I am now on break, that won't be for a while. Long story short, I am stuck with my personal email right now.
